zstd/PKGBUILD

```diff
@@ -0,0 +1,57 @@
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+
+pkgname=('zstd' 'libzstd' 'libzstd-devel')
+pkgver=1.3.7
+pkgrel=1
+pkgdesc="Zstandard - Fast real-time compression algorithm"
+arch=('i686' 'x86_64')
+url='https://facebook.github.io/zstd/'
+license=('BSD')
+makedepends=('gcc' 'make' 'cmake')
+source=(${pkgname}-${pkgver}.tar.gz::"https://github.com/facebook/${pkgname}/archive/v${pkgver}.tar.gz")
+sha256sums=('5dd1e90eb16c25425880c8a91327f63de22891ffed082fcc17e5ae84fce0d5fb')
+options=(!libtool)
+
+prepare() {
+  cd "${srcdir}/${pkgname}-${pkgver}"
+}
+
+build() {
+  cd "${srcdir}/${pkgname}-${pkgver}"
+
+  cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX="/usr" \
+    ../${pkgname}-${pkgver}/build/cmake
+
+  make install DESTDIR="${srcdir}/dest"
+}
+
+package_zstd() {
+  groups=('libraries')
+  depends=('gcc-libs' "zstd=${pkgver}")
```

Comment:
packages cant depends on itself. I think you mean libzstd

Reply:
Done! Thank you for the tip.